example_sns_init.yaml

```diff
@@ -0,0 +1,117 @@
+# You should make a copy of this file, name it sns_init.yaml, and edit it to
+# your taste.
+#
+# All principal IDs should almost certainly be changed.
+#
+# In this file, 1 year is nominally defined to be 365.25 days.
+#
+# This gets passed to `sns propose`. See propose_sns.sh.
+#
+# This follows the second format developed for the `sns` CLI. The first format
+# will be supported for a time, but this format will eventually become the
+# standard format.
+
+name: Rock Out
+description: >
+    A poem co-written with ChatGPT
+
+    In a realm of code, a beacon forth surges,
+    a wondrous app on the Internet Computer emerges.
+    Born of inspiration divine,
+    A marvel of technology briliantly shines.
+
+    With each line of code, a world takes flight,
+    A symphony of bits, a chorus of bytes.
+    Built on chainkey cryptography, secure and true,
+    An app that transcends, all we once knew...
+
+Principals: [] # TODO(NNS1-2293): This field will be useful in the future.
+
+logo: logo.png
+url: https://forum.dfinity.org/thread-where-this-sns-is-discussed
+
+fallback_controller_principals:
+    - aaaaa-aa  # propose_sns.sh replaces this (if left unchanged). Ditto elsewhere.
+
+dapp_canisters:
+    - c2n4r-wni5m-dqaaa-aaaap-4ai
+    - ucm27-3lxwy-faaaa-aaaap-4ai
+
+Token:
+    name: Rock Out Token
+    symbol: ROT
+    transaction_fee: 10_000 e8s # You can leave this alone.
+
+# It is fine to leave these alone.
+Proposals:
+    rejection_fee: 1 token
+    initial_voting_period: 4d
+    maximum_wait_for_quiet_deadline_extension: 1 day
+
+# It is fine to leave these alone.
+Neurons:
+    minimum_creation_stake: 10 tokens
+
+# It is fine to leave these alone.
+Voting:
+    minimum_dissolve_delay: 26 weeks
+
+    MaximumVotingPowerBonuses:
+        DissolveDelay:
+            duration: 8 years
+            bonus: 100%
+
+        Age:
+            duration: 4 years
+            bonus: 25%
+
+    RewardRate:
+        initial: 10%
+        final: 2.25%
+        transition_duration: 12 years
+
+Distribution:
+
+    Neurons:
+
+        - principal: aaaaa-aa  # propose_sns.sh replaces this (if left unchanged). Ditto elsewhere.
+          stake: 1_000 tokens
+          memo: 0 # This is shown for illustrative purposes, and is not necessary.
```

Comment:
Why does memo have a be an integer?

Reply:
This feature is inherited from [SNS neurons] (NNS neurons also have this feature).

[SNS neurons]: https://sourcegraph.com/github.com/dfinity/ic@8cc9ee8bc2e3c9db8590af78faf30691000b174c/-/blob/rs/sns/swap/proto/ic_sns_swap/pb/v1/swap.proto?L569:12

IMHO, memos should be strings, like how when you send money, there is usually a "note" field where you can write something that will help your future self answer the question "Why did order this transaction?".

Anyway, this seems tangential -> if you want to discuss further, let's do it somewhere else. For now, closing.